SVGKit.podspec

```diff
@@ -14,12 +16,14 @@ Pod::Spec.new do |s|
                  'Moritz Pfeiffer' => 'moritz.pfeiffer@alp-phone.ch' }
   s.source   = { :git => 'https://github.com/SVGKit/SVGKit.git', :branch => "2.x" }
 
-  s.ios.source_files = 'Source/*{.h,m}', 'Source/DOM classes/**/*.{h,m}', 'Source/Exporters/*.{h,m}', 'Source/Parsers/**/*.{h,m}', 'Source/QuartzCore additions/**/*.{h,m}', 'Source/Sources/**/*.{h,m}', 'Source/UIKit additions/**/*.{h,m}', 'Source/Unsorted/**/*.{h,m}'
-  s.tvos.deployment_target = '9.0'
+  s.source_files = 'Source/*{.h,m}', 'Source/**/*{.h,m}'
+  s.ios.exclude_files = 'Source/AppKit additions/*{.h,m}', 'Source/Exporters/SVGKExporterNSImage{.h,m}'
```

Comment:
What impact does it have in CP to exclude these files?

Reply:
This files does not availble for iOS, it's specially for Mac OS X.

CocoaPods's podspec files is a build workflow base on the expression. You express to tell which files are compiled. It's not like the GNU makefile (Which is based on the step, you write code to till what step to do).

Our Xcode project SVGKit-iOS.xcproj have the same configuration. This files are excluded from iOS/tvOS.

Reply:
See the configuration in the Xcode project.

![image](https://user-images.githubusercontent.com/6919743/47473208-4f067880-d844-11e8-8262-7a0400f82c84.png)

![image](https://user-images.githubusercontent.com/6919743/47473286-9e4ca900-d844-11e8-9bb9-6bdb2f42321a.png)

Actually, CocoaPods does not need you to configuration anything in the Xcode. All the thing is done using their syntax and human-readable expression. However, for Carthage user to manual installed user, they need Xcode configuration. So both of these two are used in the large project like our team's framework SDWebImage/CocoaLumberjack. (There are also tools to automatically sync the Xcode configuration to CocoaPods podspec)

Reply:
Sorry, I should have been explicit - it was the "AppKit additions" I was querying on that line?

Reply:
This change seems to break the build for me on iOS. SVGKExporterNSImage.h is excluded but SVGKExporterNSImage.m is not.

Reply:
Ouch. Thanks for spotting that. We're doing a final check of the build on all projects now - hopefully @dreampiggy can correct that (with a new PR / patch).